File: web/src/model/victron_controller/dashboard/from_0_2_0_actuated.ts
// @ts-nocheck
import {Actuated as v0_2_0_Actuated} from './v0_2_0/Actuated'
import {Actuated as dashboard_Actuated} from './Actuated'

export function convert__actuated__from__0_2_0(from: v0_2_0_Actuated): dashboard_Actuated {
    return new dashboard_Actuated (
        JSON.parse(JSON.stringify(from.grid_setpoint)),
        JSON.parse(JSON.stringify(from.input_current_limit)),
        JSON.parse(JSON.stringify(from.zappi_mode)),
        JSON.parse(JSON.stringify(from.eddi_mode)),
        JSON.parse(JSON.stringify(from.schedule_0)),
        JSON.parse(JSON.stringify(from.schedule_1))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_actuated.rs


pub fn convert__actuated__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::actuated::Actuated) -> crate::victron_controller::dashboard::actuated::Actuated {
    crate::victron_controller::dashboard::actuated::Actuated {
        grid_setpoint: serde_json::from_value(serde_json::to_value(&from.grid_setpoint).unwrap()).unwrap(),
        input_current_limit: serde_json::from_value(serde_json::to_value(&from.input_current_limit).unwrap()).unwrap(),
        zappi_mode: serde_json::from_value(serde_json::to_value(&from.zappi_mode).unwrap()).unwrap(),
        eddi_mode: serde_json::from_value(serde_json::to_value(&from.eddi_mode).unwrap()).unwrap(),
        schedule_0: serde_json::from_value(serde_json::to_value(&from.schedule_0).unwrap()).unwrap(),
        schedule_1: serde_json::from_value(serde_json::to_value(&from.schedule_1).unwrap()).unwrap(),
    }
}